cypress/integration/WebInterface/Test Cases/TestCaseValidations.spec.ts

```diff
@@ -68,7 +66,7 @@ describe('Test Case Validations', () => {
 
     })
 
-    it('Create Test Case: Description more than 250 characters', () => {
+    it.skip('Create Test Case: Description more than 250 characters', () => {
```

Comment:
why are we skipping this test?

Reply:
Due to the bug where the login page does not load, I would get one or two tests to run, at a time, before hitting that bug. So, to help move things along a little quicker, I would skip all of the tests that passed with each attempt to run the spec file. That means by the time I got to the end of the spec file, I had to go back and remove a lot of .skip. I must have overlooked this one, when I removed all of the ones that were not necessary.

I have removed it now, though.